src/views/Proposals.vue

```diff
@@ -150,10 +159,19 @@ export default {
     }
   },
   methods: {
-    ...mapActions(['getProposals'])
+    ...mapActions(['getProposals']),
+    checkState() {
+      const located = location.hash.replace(/^([^#]*#[^#]*#)|$/gim, "");
```

Comment:
We still need to use $router here, the interface should work if we change router mode from "hash" to "history" here: https://github.com/bonustrack/snapshot/blob/b6eb963edcd7bb69a2c219ceb6589651e1dbdd56/src/router.ts#L19 the router take care of this.

Reply:
yes, ddnt we say to go with the hash until we add router after yesterday night?

Reply:
Hm not sure what i said about it, but i'm sure we don't want the hash here

Reply:
ok